configure.ac

```diff
@@ -1844,6 +1844,9 @@ yes)
 	    ;;
 
     Darwin*)
+        # Issue #35569: Expose the RFC 3542 socket options.
+        CFLAGS="-D__APPLE_USE_RFC_3542 ${CFLAGS}"
```

Comment:
Sorry, another comment.  It's kind of ugly to have this define now show up in CFLAGS and propagate to every C build in the Makefile when it is only needed for the socket module (right?).  Perhaps it would be better to modify the top-level setup.py to add the define there in the Darwin case.

Reply:
Absolutely. I'll have a look at it. Thanks for the suggestion.

Reply:
Fixed in f6bda5d49fef82ad424e5b85e6e7fa29a2bc3cfd